devsetup/Makefile

```diff
@@ -176,11 +176,18 @@ standalone_snapshot: ## Create standalone snapshot
 	$(eval $(call vars))
 	virsh --connect=qemu:///system detach-device-alias edpm-compute-${EDPM_COMPUTE_SUFFIX} fs0 --live
```

Comment:
This line should take care of removing the virtiofs device and allow the live snapshot. At least it worked for me... :)

Reply:
Is it because we need a delay between the two to give libvirt space to detach? Is it async?

Reply:
Yes this just came up at the hackathon. I would rather not stop the standalone VM as it might not come back up 100% well, but a simple e.g. 3-5 second wait between the virtiofs detach and the snapshot should do it. The snapshot is being done so infrequently that it shouldn't matter it takes a little longer.

Reply:
(Sean mentioned the detach operation is indeed async.)

Reply:
I think this change still makes sense, as a fallback option to have at least the cold snapshot created, if taking a 3 seconds snap didn't trick the virtiofs monster

Reply:
makes sense, this indeed looks no longer needed